(*UserRepository).UpdateAccountがたまに落ちる

https://github.com/traPtitech/traPortfolio/actions/runs/5210335442/jobs/9401287645?pr=535
```text
--- FAIL: TestUserRepository_UpdateAccount (0.00s)
    --- FAIL: TestUserRepository_UpdateAccount/Success (0.01s)
        user_impl_test.go:1168: 
            	Error Trace:	/home/runner/work/traPortfolio/traPortfolio/interfaces/repository/user_impl_test.go:1168
            	Error:      	Received unexpected error:
            	            	call to ExecQuery 'UPDATE `accounts` SET `check`=?,`name`=?,`type`=?,`url`=?,`updated_at`=? WHERE `id` = ?' with args [{Name: Ordinal:1 Value:false} {Name: Ordinal:2 Value:jCC1aUPgii} {Name: Ordinal:3 Value:0} {Name: Ordinal:4 Value:https://9p6blEOlc} {Name: Ordinal:5 Value:[20](https://github.com/traPtitech/traPortfolio/actions/runs/5210258744/jobs/9401146840#step:4:21)[23](https://github.com/traPtitech/traPortfolio/actions/runs/5210258744/jobs/9401146840#step:4:24)-06-08 10:50:36.629 +0000 UTC} {Name: Ordinal:6 Value:ed815887-eadb-40a7-8d47-fd7cb64c52a6}], was not expected, next expectation is: ExpectedQuery => expecting Query, QueryContext or QueryRow which:
            	            	  - matches sql: '^SELECT \* FROM `accounts` WHERE `accounts`\.`user_id` = \? AND `accounts`\.`type` = \? ORDER BY `accounts`\.`id` LIMIT 1$'
            	            	  - is with arguments:
            	            	    0 - 9ec8cb69-0dbd-46b2-bf20-01b8[28](https://github.com/traPtitech/traPortfolio/actions/runs/5210258744/jobs/9401146840#step:4:29)b7d13f
            	            	    1 - {v:0 valid:true}
            	            	  - should return error: no rows in result set
            	Test:       	TestUserRepository_UpdateAccount/Success
FAIL
	github.com/traPtitech/traPortfolio/interfaces/repository	coverage: 96.2% of statements
FAIL	github.com/traPtitech/traPortfolio/interfaces/repository	0.917s
```

https://github.com/traPtitech/traPortfolio/blob/10ed5d68697ffeed63510a7aad41e7e71b696e49/interfaces/repository/user_impl_test.go#L1025-L1035

ここを ↓ に書き換えると`go test $(go list ./... | grep -v "integration_tests") -coverprofile=coverage_unit.txt -race -vet=off`で落ちた

```go
			name: "Success",
			args: args{
				userID:    random.UUID(),
				accountID: random.UUID(),
				args: &repository.UpdateAccountArgs{
					DisplayName: optional.From("jCC1aUPgii"),
					URL:         optional.From("https://9p6blEOlc"),
					PrPermitted: optional.From(random.Bool()),
					Type:        optional.From(domain.AccountType(0)),
				},
			},
```

```
--- FAIL: TestUserRepository_UpdateAccount (0.00s)
    --- FAIL: TestUserRepository_UpdateAccount/Success (0.01s)
        user_impl_test.go:1168: 
                Error Trace:    /path/to/traPortfolio/interfaces/repository/user_impl_test.go:1168
                Error:          Received unexpected error:
                                call to ExecQuery 'UPDATE `accounts` SET `check`=?,`name`=?,`type`=?,`url`=?,`updated_at`=? WHERE `id` = ?' with args [{Name: Ordinal:1 Value:true} {Name: Ordinal:2 Value:jCC1aUPgii} {Name: Ordinal:3 Value:0} {Name: Ordinal:4 Value:https://9p6blEOlc} {Name: Ordinal:5 Value:2023-07-20 22:54:33.843 +0900 JST} {Name: Ordinal:6 Value:6a751da3-fe5d-4b97-9877-80fb3169fa03}], was not expected, next expectation is: ExpectedQuery => expecting Query, QueryContext or QueryRow which:
                                  - matches sql: '^SELECT \* FROM `accounts` WHERE `accounts`\.`user_id` = \? AND `accounts`\.`type` = \? ORDER BY `accounts`\.`id` LIMIT 1$'
                                  - is with arguments:
                                    0 - b0c632c8-68ad-40cb-92d2-280ef545baa0
                                    1 - {v:0 valid:true}
                                  - should return error: no rows in result set
                Test:           TestUserRepository_UpdateAccount/Success
FAIL
        github.com/traPtitech/traPortfolio/interfaces/repository        coverage: 96.2% of statements
FAIL    github.com/traPtitech/traPortfolio/interfaces/repository        0.931s
```

--- a/interfaces/repository/user_impl_test.go
+++ b/interfaces/repository/user_impl_test.go
@@ -1039,7 +1039,10 @@ func TestUserRepository_UpdateAccount(t *testing.T) {
 				f.h.Mock.
 					ExpectQuery(makeSQLQueryRegexp("SELECT * FROM `accounts` WHERE `accounts`.`id` = ? AND `accounts`.`user_id` = ? ORDER BY `accounts`.`id` LIMIT 1")).
 					WithArgs(anyUUID{}, args.userID).
-					WillReturnRows(sqlmock.NewRows([]string{"id"}).AddRow(args.accountID))
+					WillReturnRows(
+						sqlmock.NewRows([]string{"id", "type", "name", "url", "check"}).
+							AddRow(args.accountID, (args.args.Type.ValueOrZero()+1)%domain.AccountLimit, args.args.DisplayName.ValueOrZero(), args.args.URL.ValueOrZero(), args.args.PrPermitted.ValueOrZero()),
+					)
 				f.h.Mock.
 					ExpectQuery(makeSQLQueryRegexp("SELECT * FROM `accounts` WHERE `accounts`.`user_id` = ? AND `accounts`.`type` = ? ORDER BY `accounts`.`id` LIMIT 1")).
 					WithArgs(args.userID, args.args.Type).

